### PR TITLE
Use only one message for NDK discovery, remove accidental newline

### PR DESCRIPTION
**Pull request by mkustermann**

### CODE REVIEW

**Review by tvolkert — APPROVED** [review]
LGTM

File: packages/flutter_tools/lib/src/android/android_workflow.dart [hunk 1]
```diff
@@ -120,13 +120,9 @@ class AndroidWorkflow extends DoctorValidator implements Workflow {
     messages.add(new ValidationMessage('Android SDK at ${androidSdk.directory}'));
 
     messages.add(new ValidationMessage(androidSdk.ndkDirectory == null
-          ? 'Unable to locate Android NDK.\n'
+          ? 'Android NDK location unknown (optional, useful for using native profiling tools)'
```
Comment (devoncarew) [inline packages/flutter_tools/lib/src/android/android_workflow.dart]: Perhaps, `Android NDK location not configured (this optional configuration is useful for native profiling support)`?

**Review by devoncarew — APPROVED** [review]
I included some possible word-smithing; lgtm.

File: packages/flutter_tools/lib/src/android/android_workflow.dart (hunk 1 above)
Comment (mkustermann) [reply packages/flutter_tools/lib/src/android/android_workflow.dart]: Done

**Pull request merged**